Jenkinsfile

```diff
@@ -140,11 +142,15 @@ spec:
     imagePullPolicy: Always
     env:
       - name: JAVA_TOOL_OPTIONS
-        value: -Xmx6G
+        value: -Xmx1G
```

Comment:
Here we are trying to change the default JVM max memory heap size from 6gb to 1gb.  We may see some OOMs issues as a result and can adjust.  I would like to identify which jvm process needs 6gb currently.